src/sentry/issues/auto_source_code_config/task.py

```diff
@@ -77,8 +79,8 @@ def process_event(project_id: int, group_id: int, event_id: str) -> list[CodeMap
         trees = get_trees_for_org(installation, org, extra)
         trees_helper = CodeMappingTreesHelper(trees)
         code_mappings = trees_helper.generate_code_mappings(frames_to_process)
-        if event.platform not in DRY_RUN_PLATFORMS:
-            set_project_codemappings(code_mappings, installation, project)
+        if platform not in DRY_RUN_PLATFORMS:
+            set_project_codemappings(code_mappings, installation, project, platform)
```

Comment:
`platform` will be used as a tag for the metric.
<img width="464" alt="image" src="https://github.com/user-attachments/assets/9c5a5e08-8eca-406c-a4c1-0d2a2383c28f" />